fidesctl/src/fidesctl/core/api_helpers.py

```diff
@@ -67,3 +67,30 @@ def get_server_resource(
     )
 
     return server_resource
+
+
+def list_server_resources(
+    url: str,
+    resource_type: str,
+    headers: Dict[str, str],
+) -> List[FidesModel]:
+    """
+    Get a list of resources from the server and return them as parsed objects.
+
+    Throws an error if the response is empty.
```

Comment:
Does this still throw an error if the response is empty like when we finished up yseterday? Looks like it might return an empty list at this point so we may want to update the comment

Reply:
added tests to clarify this